02_bubble/brvishnu92/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+
+	swap := true
+	for swap {
+		z := s[0]
+		swap = false
+		for i, x := range s {
+			if x < z && i != 0 {
+				swap = true
+				s[i-1], s[i] = s[i], s[i-1]
+			}
+			z = x
+		}
+	}
+	return s
```

Comment:
Does produce a sorted copy or mutate the original array?

Reply:
Yeah i was mutating it.. My bad... Thanks :-)